README.md

```diff
@@ -100,6 +100,15 @@ For protobuf-based codegen integrated with the Maven build system, you can use
         </execution>
       </executions>
     </plugin>
+    <plugin>
+			<groupId>org.apache.maven.plugins</groupId>
```

Comment:
We don't really include this part of the snippet, because it isn't grpc-specific or strictly necessary. We want to keep the snippet short and let people manage their pom.xml however they normally would.